ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala

```diff
@@ -581,14 +579,6 @@ object CliConfig {
         .action((acsContractFetchingParallelism, config) =>
           config.copy(acsContractFetchingParallelism = acsContractFetchingParallelism)
         ),
-      opt[Int]("acs-global-parallelism-limit")
```

Comment:
This should be deprecated and ignored instead

Reply:
Deprecated it by mentioning it in the option's text